cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -872,6 +872,14 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
                     )
                 ],
             ),
+            iam.PolicyStatement(
+                sid="CloudFormation",
+                actions=[
+                    "cloudformation:SignalResource",
+                ],
+                effect=iam.Effect.ALLOW,
+                resources=["*"],
```

Comment:
A wildcard resource is not a good solution because it makes the policy too permissive. Try restricting the permissions as much as possible.

Reply:
Done!